[SYNTH-12978] Clarify use of global configuration file in README (#1368)
DataDog/datadog-ci: src/commands/synthetics/README.md

### Global configuration file options

> Create a JSON configuration file on your system. Specify the path to the file using the `--config` flag [when launching your tests](#run-tests). If you don't specify a file path, Datadog looks for a file with the default filename of `datadog-ci.json`.

Reviewer: I see this text is exactly the same as line 53. Could you change that bullet point `3.`  to only redirect to the `#global-configuration-file-options` section?

Also, I think the paragraph could use a little introduction, instead of starting with "Create a JSON configuration file" out of nowhere

Author: Ah yeah this is a side effect of splitting the changes into multiple PRs 😅 on my other branch i've reordered the points and slightly changed them, removing this part from point `3.` 

https://github.com/DataDog/datadog-ci/pull/1372/files#diff-5a0fda3a46d1c105f78a54a8d87902bdb9886b1739928f72abc7cefc23a2f86bR38-R62

### API

> By default, `datadog-ci` runs at the root of the working directory and looks for `{,!(node_modules)/**/}*.synthetics.json` files (every file ending with `.synthetics.json`, except for those in the `node_modules` folder) to find a Test Config file.

Reviewer: "Test Config" file is what we use internally, for short
```suggestion
By default, `datadog-ci` runs at the root of the working directory and looks for `{,!(node_modules)/**/}*.synthetics.json` files (every file ending with `.synthetics.json`, except for those in the `node_modules` folder) to find a [test configuration file](#test-files).
```

> The default value for the Global Configuration file is `datadog-ci.json`. If you name you global config file that you don't need to use the `--config` flag, the file will be automatically picked up.

Reviewer 2: ```suggestion
The default value for the Global Configuration file is `datadog-ci.json`. If you use this name for your global config file, you don't need to use the `--config` flag, as the file is automatically picked up.
```

Reviewer: here would be my suggestion if we are really going to keep that sentence - though I think we are just explaining the concept of a default value 😅 
```suggestion
The default file name for the [global configuration file](#global-configuration-file-options) is `datadog-ci.json`. If you use this name for your global configuration file, you may omit the `--config` flag.
```

Reviewer 2: @Drarig29 thanks, I agree that your suggestion is better in this case (if the decision is to keep the sentence).

> The default file name for the [global configuration file](#global-configuration-file-options) is `datadog-ci.json`. If you use this name for your global configuration file, you may omit the `--config` flag.

Reviewer: @buraizu do you prefer "file name" or "filename"? My suggestion here is just to align with the spelling in other places of that file
```suggestion
The default filename for the [global configuration file](#global-configuration-file-options) is `datadog-ci.json`. If you use this name for your global configuration file, you may omit the `--config` flag.
```

Reviewer 2: @Drarig29 I see that both styles are used about equally in the documentation currently, even within this page, so either should be fine.

}
```

> They can also be added the same global configuration file as used for the run-tests command.

Reviewer 2: ```suggestion
These options can also be added to the same global configuration file used for the run-tests command.
```

Pass this config file to the command with the `--config` flag:

```bash
datadog-ci synthetics upload-application --config global-config.json
```

> The default value for the Global Configuration file is `datadog-ci.json`. If you name you global config file that you don't need to use the `--config` flag, the file will be automatically picked up.

Reviewer 2: ```suggestion
The default value for the Global Configuration file is `datadog-ci.json`. If you use this name for your global config file, you don't need to use the `--config` flag, as the file is automatically picked up.
```

Reviewer: ditto
